packages/experimental-nextjs-app-support/README.md

```diff
@@ -154,6 +158,60 @@ export default function RootLayout({
 
 If you want to make the most of the streaming SSR features offered by React & the Next.js App Router, consider using the [`useSuspenseQuery`](https://www.apollographql.com/docs/react/api/react/hooks-experimental/#using-usesuspensequery_experimental) and [`useFragment`](https://www.apollographql.com/docs/react/api/react/hooks-experimental/#using-usefragment_experimental) hooks.
 
+### Preloading data in RSC for usage in Client Components
+
+You can also preload data in RSC to populate the cache of your Client Components.
+
+For that, follow the setup steps for both RSC and Client Components as laid out in the last two paragraphs.
+
+Then you can use the `PreloadQuery` component in your React Server Components:
+
+```jsx
+<PreloadQuery
+  query={QUERY}
+  variables={{
+    foo: 1,
+  }}
+>
+  <Suspense fallback={<>loading</>}>
+    <ClientChild />
```

Comment:
I think it might be a good idea to show what `ClientChild` looks like here. Its not obvious how `PreloadQuery` relates to what is used in `ClientChild` without it. I assume that `ClientChild` would probably contain a `useSuspenseQuery`, but I can only guess that because I've followed this work for some time. Since you detail it in the paragraph below, I'd show the implementation of it here to really hammer home how `PreloadQuery` and `useSuspenseQuery` work together :)

Reply:
0db028488ac7b53825ccae4470285f2b32fe72be